src/CLI/clitree/cli-xml/vrf.xml

```diff
@@ -25,12 +25,12 @@ limitations under the License.
     <VIEW name="enable-view">
       <COMMAND name="show ip vrf" help="Show vrf information">
```

Comment:
Can we capitalize "VRF" in the `help` string?

Reply:
Done